src/Icons/doc/index.rst

```diff
@@ -83,10 +166,40 @@ a subdirectory, the *name* will be ``subdirectory:icon_name``.
 
 On-Demand Open Source Icons
```

Comment:
Should this title be removed?

Reply:
Probably... and i realize i once again failed my rebase 😮‍💨

Reply:
Good (i think)